### PR TITLE
Smart renewal

### PR DESCRIPTION
**Pull request by hansfilipelo**
This renewal script checks whether a certificate needs renewal or not before renewing it. If there's no need to renew, renewal will not be done.

### CODE REVIEW

File: update/renewCerts.bash [hunk 1]
```diff
@@ -6,11 +6,29 @@ cd $scriptDir
...
 letsencFolder=/etc/letsencrypt
+dateNow=$(date -d "now" +%s)
+
 
 # Update all certs with cli.ini file in folder for Let's Encrypt
 for configFile in $(ls $letsencFolder/*.cli.ini); do
-	echo "Renewing cert using $configFile"
-	../letsencrypt/letsencrypt-auto certonly --renew-by-default --config $configFile
+	
+	primaryDomain=`grep "^\s*domains" $configFile | sed "s/^\s*domains\s*=\s*//" | sed 's/(\s*)\|,.*$//'`
```
Comment (Rovanion) [inline update/renewCerts.bash]: Kommer den här att matcha alla domän som ett cert är giltigt för?

Comment (hansfilipelo) [inline update/renewCerts.bash]: This will only match the first domain in the list (which is necessary for this script to work).

Comment (hansfilipelo) [inline update/renewCerts.bash]: For clarity: The cert is symlinked to the folder /etc/letsencrypt/live/DOMAIN where DOMAIN is the first domain in the list from the given .ini-file. The certificate itself can however be valid for a long list of domains.

Comment (Rovanion) [inline update/renewCerts.bash]: Okey. So it matches only DOMAIN but refreshes the entire cert with all the domains it's valid for.

Comment (hansfilipelo) [inline update/renewCerts.bash]: Yupp. It refreshes the certificate according to the .ini file (which includes all domains).

**Comment by Rovanion** [comment]
Tested on webtest and it worked for the two sites installed there. So I'm going to go ahead and merge this!

**Pull request merged**